examples/python/plots/README.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: Plots
 python: https://github.com/rerun-io/rerun/tree/latest/examples/python/plots/main.py
-thumbnail: https://static.rerun.io/ca0c72df93d70c79b0e640fb4b7c33cdc0bfe5f4_plots_480w.png
-description: "Demonstration of various plots and charts supported by Rerun."
 tags: [2d, plots, api-example]
+description: "Demonstration of various plots and charts supported by Rerun."
+thumbnail: https://static.rerun.io/ca0c72df93d70c79b0e640fb4b7c33cdc0bfe5f4_plots_480w.png
+thumbnail_dimensions: [480, 271]
+build_args: []
```

Comment:
This line looks like it is redundant - surely the default is an empty list, so I can just remove it from the file, right?

Well, that is very reasonable, unless you very recently read this comment at the top of `crates/re_viewer/build.rs`:

> …only includes those with a `build_args` set in their `README.md` frontmatter.

Oh shit, `build_args` is load bearing!?

Much safer to make this explcit:

```suggestion
include_on_welcome_page: true
```